<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="sidebar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>250</width>
      <height>768</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <widget class="QPushButton" name="homeButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="worksButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Works</string>
     </property>
    </widget>
    <widget class="QPushButton" name="artistsButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>260</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Artists</string>
     </property>
    </widget>
    <widget class="QPushButton" name="museumsButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>330</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Museums</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>1116</width>
      <height>768</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="homePage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>390</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Painting of the Day</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>390</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Statistics</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="worksPage">
     <widget class="QLineEdit" name="worksPageInput">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>351</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="listAllWorksButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>List of all Paintings</string>
      </property>
     </widget>
     <widget class="QPushButton" name="workInfoButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>190</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Painting Information</string>
      </property>
     </widget>
     <widget class="QPushButton" name="artistsNationalitiesButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>190</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Artists' Nationalities</string>
      </property>
     </widget>
     <widget class="QPushButton" name="workInMuseumButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>270</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Museums w/ this Painting</string>
      </property>
     </widget>
     <widget class="QPushButton" name="paintingSizeInfoButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>270</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Painting Size Information</string>
      </property>
     </widget>
     <widget class="QPushButton" name="paintingPriceButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>270</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Painting Price Information</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="workTableWidget">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>400</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="artistsPage">
     <widget class="QLineEdit" name="artistPageInput">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>351</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="listAllArtistsButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>List of all Artists</string>
      </property>
     </widget>
     <widget class="QPushButton" name="artistInfoButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>190</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Artist Information</string>
      </property>
     </widget>
     <widget class="QPushButton" name="artistsNationalitiesButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>190</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Artists' Nationalities</string>
      </property>
     </widget>
     <widget class="QPushButton" name="allArtistsWorksButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>280</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>All Works by This Artist</string>
      </property>
     </widget>
     <widget class="QPushButton" name="artistStylesButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Artist's Styles</string>
      </property>
     </widget>
     <widget class="QPushButton" name="artistWorkPricesButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>280</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Prices of this Artist's Work</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>410</y>
        <width>621</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="museumsPage">
     <widget class="QLineEdit" name="museumsPageInput">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>351</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="listAllMuseumsButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>List of all Museums</string>
      </property>
     </widget>
     <widget class="QPushButton" name="museumInfoButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>190</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Museum Information</string>
      </property>
     </widget>
     <widget class="QPushButton" name="artistsInMuseumButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>190</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Artists in a Museum</string>
      </property>
     </widget>
     <widget class="QPushButton" name="worksInMuseumButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>270</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Works in a Museum</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addressInfoButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>270</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>All Addresses</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="museumTableWidget">
      <property name="geometry">
       <rect>
        <x>225</x>
        <y>460</y>
        <width>581</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="museumDaysAndHoursButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>270</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Museum Days/Times</string>
      </property>
     </widget>
     <widget class="QPushButton" name="museumAddressButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>350</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Museum Address</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
